Microsoft.Toolkit.Uwp.SampleApp/SamplePages/HamburgerMenu/HamburgerMenuCode.bind

```diff
@@ -53,7 +53,7 @@
                                 CompactPaneLength="@[CompactPaneLength:Slider:48:10-80]"
                                 HamburgerHeight="@[HamburgerHeight:Slider:48:10-80]"
                                 HamburgerVisibility="@[HamburgerVisibility:Enum:Visibility.Visible]"
-                                IsPaneOpen="@[IsPaneOpen:Bool:False]">
+                                IsPaneOpen="@[IsPaneOpen:Bool:False]@">
```

Comment:
There is an additional @ here

Reply:
This is a new syntax, the end @ signifies that the template should generate it using a TwoWay Binding instead of a OneWay Binding.